server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -114,6 +116,7 @@ export class ScribeLambda implements IPartitionLambda {
 		private readonly kafkaCheckpointOnReprocessingOp: boolean,
 		private readonly isEphemeralContainer: boolean,
 		private readonly localCheckpointEnabled: boolean,
+		private readonly maxLogtailLength: number,
```

Comment:
This constructor is used in `memory-orderer`, so that needs to be updated too

Reply:
Thanks @znewton , this makes sense.

I am thinking a much-simplified version might just use `this.pendingMessages` when write service or client summary. Do you have a good person for me to confirm it?

Reply:
As discussed directly on teams, will think about using `this.pendingMessages` in a later version:
> The current change feels like a safe, logical alteration of the current logic. This other pendingMessages change feels like a larger switch. Maybe move forward with the current PR to see how well it solves the problem in Prod, then open up a follow-up PR with the pendingMessages version?